.github/ci-heapster.sh

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env bash
+set -Eeuxo pipefail
+
+[[ "$RUNNER_OS" == 'Windows' ]] && IS_WIN=true || IS_WIN=false
+BIN=bin
+EXT=""
+$IS_WIN && EXT=".exe"
+mkdir -p "$BIN"
+
+is_exe() { [[ -x "$1/$2$EXT" ]] || command -v "$2" > /dev/null 2>&1; }
```

Comment:
Shouldn't `$BIN` be prefixed to `$PATH` globally so that we can find/execute `$2` if it is in `$BIN`?  In that scenario, the first test here isn't needed, just the `command -v` check.